pkg/evpn/svi.go

```diff
@@ -130,51 +164,51 @@ func (s *Server) DeleteTunnel(_ context.Context, in *pb.DeleteTunnelRequest) (*e
 		return nil, err
 	}
 	resourceID := path.Base(obj.Name)
-	// use netlink to find Subnet/Bridge
-	vrf, err := netlink.LinkByName(resourceID)
+	// use netlink to find vlan
+	vlan, err := netlink.LinkByName(resourceID)
```

Comment:
maybe vlandev or svidev would be more proper because you actually delete a vlan subinterface not just a vlan